src/openapi/streaming/streaming.ts

```diff
@@ -689,6 +701,17 @@ class Streaming extends MicroEmitter<EmittedEvents> {
         }
     }
 
+    /**
+     * Handles probe control messages and calls given callbacks at creation time
+     * 
+     * @param message - the message from server

```

Comment:
We have a lot of low/zero-value JSDoc in this project (so it looks like a convention which I guess you followed), but we tend to avoid adding more such ones